internal/namespaces/config/commands_test.go

```diff
@@ -210,6 +210,51 @@ func Test_ConfigDumpCommand(t *testing.T) {
 	}))
 }
 
+func Test_ConfigDestroyCommand(t *testing.T) {
+	path := "/tmp/test_config_destroy/"
```

Comment:
Use `TmpHomeDir`

Reply:
For the TmpHomeDir @yfodil  prefer to create my own temporary directory where I can set a path that @yfodil  define, using TmpHomeDir the folders in local and on the CI will not be the same so the Golden will not be valid: